docs/useful-tips/transform-function-and-its-parameters.md

```diff
@@ -130,17 +130,19 @@ const result2 = image.transform(rotationMatrix, {
 });
 ```
 
-As you can see using the same matrix images the image seems to be turned into opposite directions. Basically, with `inverse` set to `true` you kind of "revert" an image back to the states that it was before the matrix transformed.
+As you can see using the same matrix images the image seems to be turned into opposite directions.
 
 ![inverse test](./images/transform-parameters/inverseTest.svg)
 
 #### Full Image
 
-`fullImage` ensures that the output image is large enough to contain all transformed pixels from the source image, preventing any cropping. When true, the function automatically calculates the required output dimensions.
+`fullImage` ensures that the output image is large enough to contain all transformed pixels from the source image, preventing any cropping. When true, the function automatically calculates the required output dimensions. When false, source image dimensions are taken.
```

Comment:
I would rather say that when `true`, `width` and `height` are ignored. Also "ensuring the image is large enough" is not very accurate. It is not the same as deciding of the output dimensions based on the transformation.

Reply:
Yeah, it's better to take explanation from the docs, that it "bypasses width and height to include every pixel of the original image inside the trasnformed image".